litmus-portal/frontend/src/components/SettingsComponents/UserManagementTab/UserManagement/index.tsx

```diff
@@ -0,0 +1,353 @@
+/* eslint-disable camelcase */
+import {
+  createStyles,
+  FormControl,
+  IconButton,
+  InputAdornment,
+  Menu,
+  MenuItem,
+  Paper,
+  Select,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TextField,
+  Theme,
+  Toolbar,
+  Typography,
+  withStyles,
+} from '@material-ui/core';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import SearchIcon from '@material-ui/icons/Search';
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { RootState } from '../../../../redux/reducers';
+import ButtonFilled from '../../../Button/ButtonFilled';
+import CreateUser from '../CreateUser';
+import EditUser from '../EditUser';
+import useStyles from './styles';
+
+// StyledTableCell used to create custom table cell
+const StyledTableCell = withStyles((theme: Theme) =>
```

Comment:
Could this be shifted in `styles.ts`!

Reply:
This is custom styles, I think it's okay to be there. cc: @arkajyotiMukherjee